plugins/akka-http/src/main/java/com/navercorp/pinpoint/plugin/akka/http/interceptor/ExecutionDirectivesHandleExceptionsInterceptor.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright 2017 NAVER Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.navercorp.pinpoint.plugin.akka.http.interceptor;
+
+import akka.http.javadsl.model.HttpRequest;
+import akka.http.scaladsl.server.RequestContextImpl;
+import com.navercorp.pinpoint.bootstrap.async.AsyncContextAccessor;
+import com.navercorp.pinpoint.bootstrap.context.*;
+import com.navercorp.pinpoint.bootstrap.interceptor.AroundInterceptor;
+import com.navercorp.pinpoint.bootstrap.logging.PLogger;
+import com.navercorp.pinpoint.bootstrap.logging.PLoggerFactory;
+import com.navercorp.pinpoint.bootstrap.plugin.proxy.ProxyHttpHeaderHandler;
+import com.navercorp.pinpoint.bootstrap.plugin.proxy.ProxyHttpHeaderRecorder;
+import com.navercorp.pinpoint.plugin.akka.http.AkkaHttpConstants;
+
+import java.util.Optional;
+
+public class ExecutionDirectivesHandleExceptionsInterceptor implements AroundInterceptor {
+
```

Comment:
As-is
```
class : akka.http.scaladsl.server.directives.ExecutionDirectives, 
method : $anonfun$handleExceptions$2
```

I think It is a better way.
```
class : akka.http.scaladsl.server.directives.BasicDirectives
method : $anonfun$mapRequestContext$2
```
the difference between the two may not be so great,
the methodName is more clearness.
Since BasicDirectives is a trait class, there is likely to be less chance of future changes.